update tiup doc (#15374)
pingcap/docs: tiup/tiup-component-management.md

- status: Checks the status of a running component.
- clean: Cleans up the instance on which a component is deployed.
- help: Prints the help information. If you append another TiUP command to this command, the usage of the appended command is printed.

> - link: Link the binary of the component to the executable directory (`$TIUP_HOME/bin/`).
> - unlink: Delete the soft link generated by the `tiup link` command.

Reviewer: ```suggestion
- link: Links the binary of the component to the executable directory (`$TIUP_HOME/bin/`).
- unlink: Deletes the soft link generated by the `tiup link` command.
```

```shell
tiup uninstall --all
```

> ### Link Component

Reviewer: ```suggestion
### Link components
```

> TiUP v1.13.0 added experimental `link` and `unlink` commands to link the binary of a component to the executable directory (`$TIUP_HOME/bin/`) and remove the link. This feature allows users to call the component without going through TiUP each time, while preserving the ability to switch between multiple versions. However, this method lacks the process of automatic update checking and setting certain environment variables. Therefore, it's recommended to use it only when necessary.

Reviewer: ```suggestion
TiUP v1.13.0 adds the experimental `link` and `unlink` commands to link the binary of a component to the executable directory (`$TIUP_HOME/bin/`) and remove the link. This feature allows you to call a component without going through TiUP each time, while preserving the ability to switch between multiple versions. However, this method lacks the process of automatic update checking and setting certain environment variables and some components (for example, ctl) can not be used. Therefore, it is recommended to use it only when necessary.
```

Example 1: Install and link the latest version of the cluster component

```shell
tiup install cluster
tiup link cluster
```

Example 2: Switch the cluster component to version v1.13.0

```shell
tiup link cluster:v1.13.0
```

> The `tiup link cluster` command will output the following:

Reviewer: ```suggestion
The following is the output of the `tiup link cluster` command:
```

```shell
package cluster provides these executables: tiup-cluster
```

This indicates that the binary name of the cluster component is `tiup-cluster`. After the link command is completed, you can use the cluster component by directly typing `tiup-cluster` into the command line.

Example 3: Cancel the link of the cluster component

```shell
tiup unlink cluster
```

> Example 4: Manage the version of TiUP itself

Reviewer: ```suggestion
Example 4: Manage the version of TiUP
```

> Before v1.13.0, TiUP itself was installed in `~/.tiup/bin/` and could not coexist with multiple versions. Starting from v1.13.0, TiUP can be installed and linked like all other components.

Reviewer: ```suggestion
Before v1.13.0, TiUP is installed in `~/.tiup/bin/` and can not coexist with multiple versions. Starting from v1.13.0, You can install and link TiUP like any other components.
```

> Before v1.13.0, TiUP is installed in `~/.tiup/bin/` and can not coexist with multiple versions. Starting from v1.13.0, You can install and link TiUP like any other components.

Reviewer: ```suggestion
Before v1.13.0, TiUP is installed in `~/.tiup/bin/` and different versions can not coexist. Starting from v1.13.0, you can install and link TiUP like any other components.
```
